.github/workflows/build.yml

```diff
@@ -18,33 +18,16 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
-    - name: Install Ubuntu dependencies
-      run: >
-        sudo apt-get update -q && sudo apt-get install
-        --no-install-recommends -y xvfb python3-dev python3-gi
-        python3-gi-cairo gir1.2-gtk-3.0 libgirepository1.0-dev libcairo2-dev
-        intltool enchant python3-enchant gir1.2-poppler-0.18 python3-gst-1.0
-
-    - name: Install dependencies
+    - name: Setup build environment
       run: |
-        python3 -m pip install --upgrade pip setuptools
-        pip3 install --upgrade keyrings.alt
-
-    - name: Create Internationalization
-      run: |
-          python3 setup.py build_i18n
-          find . | grep gourmet-plugin
-
-    - name: Install
-      run: |
-        pip3 install .[epub-export,mycookbook,pdf-export,spellcheck,web-import]
+        sudo apt-get update -q
+        sudo apt-get install python3-testresources intltool
```

Comment:
What is `python3-testresources` used for?

Reply:
We were getting an error in the builds when invoking `pip`:

`ERROR: launchpadlib 1.10.13 requires testresources, which is not installed.`

It didn't seem to be otherwise impacting the build, but installing `python3-testresources` eliminates the error.